### DISCUSSION TITLE
Interior Database

### DISCUSSION THREAD

**Discussion by erox455** [shun126/UE5-DungeonGeneratorDemo#12] [Ideas]
Pls add a "Spawn on client" checkbox to the interior database. So we can spawn items only on the server for correct replication

**Comment by shun126** [comment]
Interior actors are now required to replicate.
If the load is high, it can be selected at a later date.

---
インテリアアクターはレプリケーション必須にしました。
負荷が高い場合は後日選択できるようにします。

**Reply by shun126** [reply]
from version 1.6.2, you will be able to choose between two options for spawning interior actors: server-only or client and server.
